core/examples/luigi-example-js/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "luigi-example-js",
+  "version": "0.1.0",
+  "description": "",
+  "main": "public/index.html",
+  "scripts": {
+    "build": "webpack --entry ./luigi-config.js -o ./public/luigi-config.js --mode production",
+    "start": "live-server --entry-file=index.html public",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "license": "ISC",
+  "dependencies": {
+    "@babel/core": "^7.8.3",
+    "@babel/preset-env": "^7.8.3",
+    "@kyma-project/luigi-client": "^0.7.3",
+    "@kyma-project/luigi-core": "^0.7.3",
+    "babel-loader": "^8.0.6",
```

Comment:
```suggestion

```

Reply:
As we care about bundle sizes and don't use it here, make sense to remove it and free up space.